src/emulator/extensions/utils.ts

```diff
@@ -0,0 +1,44 @@
+import * as planner from "../../deploy/extensions/planner";
+import { check } from "../../ensureApiEnabled";
+
+const EMULATED_APIS = [
+  "storage-component.googleapis.com",
+  "firestore.googleapis.com",
+  "pubsub.googleapis.com",
+  "identitytoolkit.googleapis.com",
+  // TODO: Is there a RTDB API we need to add here? I couldn't find one.
+];
+
+type UnemulatedAPI = {
```

Comment:
how about `emulatableAPI` or something similar? The type doesn't have to be specific to unemulated APIs

Reply:
Swapped this to the more generic APIInfo